components/SignIn.tsx

```diff
@@ -0,0 +1,53 @@
+import tw from 'twrnc';
+import { Link } from 'expo-router';
+import { Text, TextInput, View, Pressable } from 'react-native';
+import React from 'react';
+
+function SignIn() {
+  const [email, setEmail] = React.useState('');
+  const [password, setPassword] = React.useState('');
+
+  const inputStyle = tw`bg-white px-3.5 py-3 rounded-2xl`;
+
+  const callApi = () => {
+    console.log('Email: ' + email);
+    console.log('Password: ' + password);
+
+    setEmail('');
+    setPassword('');
+  };
+
+  return (
+    <>
+      <View style={tw`px-4`}>
+        <Text style={tw`mt-19 mb-12 text-3xl font-black`}>Sign in</Text>
```

Comment:
`<Stack.Screen />` pentru a configura header-ul, pus deasupra tuturor componentelor separat de ele cu parintele `<></>`, in pagina de login din `app/`.